js/site.js

```diff
@@ -128,16 +128,17 @@ $("#collect-data-done").click(function() {
     location.hash = '#done';
 
     var note_body =
-        "onosm.org submitted note from a business:\n" +
-        "name: " + $("#name").val() + "\n" +
-        "phone: " + $("#phone").val() + "\n" +
-        "website: " + $("#website").val() + "\n" +
-        "twitter: " + $("#twitter").val() + "\n" +
-        "facebook: " + $("#facebook").val() + "\n" +
-        "email: " + $("#email").val() + "\n" +
-        "hours: " + $("#opening_hours").val() + "\n" +
-        "category: " + $("#category").val().join(", ") + "\n" +
-        "address: " + $("#address").val(),
+        "onosm.org submitted note from a business:\n\n" +
+        "name=" + $("#name").val() + "\n" +
+        "website=" + $("#website").val() + "\n" +
+        "phone=" + $("#phone").val() + "\n" +
```

Comment:
By the way, this field probably requires the reviewer’s attention too, because the country and area code may be missing: #94. That said, phone numbers needing additional formatting are quite common in OSM anyways.

Reply:
Yeah, there are a bunch of formatting adjustments that ideally would be done in the onosm code at the time the data was submitted -- things like: "Did you forget the area code?" and converting "@blahblah" into "https://twitter.com/blahblah". But as you said, mis-formatted values are pretty common, so it seemed safe enough to allow for copy/paste for them.

Reply:
This is why I have generally been opposed to direct "copy paste" support. The values are almost never directly pastable, and I don't want to encourage adding bad data to OSM. If a mapper doesn't want to take the time to research a submission from onosm with first-hand data, they should leave it as-is and move on.

Reply:
The name value is nearly always copy-pastable, as is the website, and in most cases, the twitter and facebook. Researching is certainly needed, but not to replace these values, but to confirm the existence and location of the business. And people will still copy paste even if you use colons rather than equals signs, it'll just add a bit of extra work for no purpose.